conductor_api/src/conductor/base.rs

```diff
@@ -629,6 +629,62 @@ impl Conductor {
                     ))
                 })?;
 
+                // This is where we are checking the consistency between DNAs: for now we compare
+                // the hash provided in the TOML Conductor config file with the computed hash of
+                // the loaded dna.
+                {
+                    let dna_hash_from_conductor_config = HashString::from({
+                        match dna_config.hash {
+                            Some(v) => v,
+                            None => {
+                                // Let's warn the user that we encountered an empty DNA hash in the Conductor
+                                // config that is meant to be skipped by the consistency checking mechanisim
+                                context.log("warn/Conductor: Invalid DNA hash encountered in the Conductor configuration.");
+                                String::from("")
```

Comment:
Could we use let dna_hash_from_conductor_config = HashString::from({dna_config.hash.unwrap_or_default())}).to_owned() ? Could be a one liner

Reply:
Wouldn't that preclude issuing the logging warning?

Reply:
I guess there is a context.log warn there.  dna_config.hash.unwrap_or_else(|_| context.log;String::default()); ? what do you think?

Reply:
I have a question here, how is a empty string causing the integrity check to be skipped? I don't see empty string being treated as a special case. And if it were, I would recommend keeping it as an `Option`

Reply:
There is no skipping anymore, we found a better solution, using the appropriate DNA loader :)

Thanks @AshantiMutinta for your suggestion, I took care of it :+1: